src/components/groups/token.tsx

```diff
@@ -15,50 +15,54 @@ export const Token = component$<TokenProps>((props) => {
   return (
     <>
       <div
-        class="grid max-h-[56px] items-center gap-[8px] text-nowrap px-[20px]"
-        style="grid-template-columns: minmax(200px, 400px) minmax(100px, 200px) repeat(4, minmax(145px, 300px)) 16px;"
+        class="custom-border-1-b grid items-center text-nowrap py-[6px] text-[12px]"
+        style="grid-template-columns: minmax(200px, 400px) repeat(2, minmax(100px, 200px)) minmax(180px, 300px) repeat(2, minmax(145px, 300px)) 40px;"
       >
         <div class="flex h-[40px] items-center gap-[6px]">
-          <div class="flex min-h-[32px] min-w-[32px] items-center justify-center rounded-full border border-[#E6E6E6]">
+          <div class="custom-border-1 flex items-center justify-center rounded-[8px] p-[8px]">
             {props.icon && <img src={props.icon} width="20" height="20" />}
           </div>
           <div class="flex h-full items-center gap-[6px] overflow-x-auto">
             <p>{props.name}</p>
-            <span class="text-[10px] text-[#222222] text-opacity-[50%]">
-              {props.symbol}
-            </span>
+            <span class="custom-text-50">{props.symbol}</span>
           </div>
         </div>
-        <p class="flex h-full items-center overflow-x-auto">{props.quantity}</p>
-        <p class="font-mediumh-full flex items-center overflow-x-auto">
-          {props.value}
-        </p>
-        <p class="flex h-full items-center overflow-x-auto"></p>
-        <p class="flex h-full items-center overflow-x-auto">{props.wallet}</p>
-        <p class="flex h-full items-center overflow-x-auto underline underline-offset-2">
-          {props.network}
-        </p>
+        <div class="flex h-full items-center overflow-auto">
+          <span class="custom-border-1 rounded-[8px] p-[8px]">
+            {props.quantity}
+          </span>
+        </div>
+        <div class="flex h-full items-center overflow-auto">
+          <span class="custom-border-1 rounded-[8px] p-[8px]">
+            {props.value}
+          </span>
+        </div>
+
+        <div class="flex h-full items-center overflow-auto">
+          <span class="rounded-[8px] p-[8px]"></span>
+        </div>
+        <div class="flex h-full items-center overflow-auto">
+          <span class="custom-border-1 rounded-[8px] p-[8px]">
+            {props.wallet}
+          </span>
+        </div>
+        <div class="flex h-full items-center overflow-auto font-medium">
+          <span class="custom-border-1 rounded-[8px] p-[8px] ">
+            {props.network}
+          </span>
+        </div>
         <button class="rounded-[2px] bg-[#22222214] bg-opacity-[8]">
           <svg
-            width="20"
-            height="20"
-            viewBox="0 0 20 20"
-            fill="none"
             xmlns="http://www.w3.org/2000/svg"
+            width="4"
+            height="12"
+            viewBox="0 0 4 12"
+            fill="none"
             onClick$={props.onClick$}
           >
```

Comment:
I believe that button is the one that shall have `onClick$={props.onClick$}` isntead of svg, besides shouldn't we consider centering svg inside the button?
Also IMHO button is to wide rn